### ISSUE TITLE
Better explain what this module can do in the README

### ISSUE THREAD

**Issue by indigoxela** [backdrop-contrib/mini_layouts#27]
Currently there's "Provide blocks which allow positioning content within them in layouts."

I don't think this really explains the power of this module. As a module maintainer I'm also always struggling with the module description and / or documentation, so maybe we find a volunteer to extend the description.

Possibly @stpaultim? :wink:

**Comment by stpaultim** [comment]
Good idea. I'll try to help with this after this weekend. Focused on core issues today.

**Comment by ericfoy** [comment]
Here's the  best description I can think of:
"Provides the ability to **nest layouts**. This module presents a Mini Layout as a block which may be dropped into a standard layout."
